modes.js

```diff
@@ -43,14 +42,14 @@ export async function appendMode(commenter, identifier, message) {
   console.log(`Checking if a comment already exists for ${identifier}.`);
   const matchingComment = await findMatchingComment(commenter, identifier);
 
-  const comment = `${matchingComment.body}\n${message}`;
-
   if (matchingComment) {
     console.log(`Updating an existing comment for ${identifier}.`);
+    const comment = `${matchingComment.body}\n${message}`;
```

Comment:
should we add the identifier prefix here too?

Reply:
nope. Because the matching comment would already have the identifier prefix.